doc/source/reference/xgboost/sklearn.rst

```diff
@@ -0,0 +1,65 @@
+.. _api.sklearn:
+
+================
+Scikit-Learn API
+================
+
+XGBRegressor
+============
+.. currentmodule:: xorbits.xgboost
+
+Constructor
+~~~~~~~~~~~
+.. autosummary::
+   :toctree: generated/
+
+   XGBRegressor
+
+Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Better make the length of the title and the underline the same to prevent the following warning:

`Title underline too short/long.`